src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -614,12 +619,22 @@
         {
             "name": "codewhisperer_codeScanIssueHover",
             "description": "Called when a code scan issue is hovered over",
-            "metadata": [{ "type": "findingId" }, { "type": "detectorId" }]
+            "metadata": [{ "type": "findingId" }, { "type": "detectorId" }, { "type": "ruleId", "required": false }]
         },
         {
             "name": "codewhisperer_codeScanIssueApplyFix",
```

Comment:
the feature was released. these metrics should be upstreamed to https://github.com/aws/aws-toolkit-common/blob/main/telemetry/definitions/commonDefinitions.json

Reply:
Sure, moving it to the common package